.github/workflows/ci.yaml

```diff
@@ -102,3 +102,22 @@ jobs:
         run: |
           cargo clippy --manifest-path ${{ matrix.crates }} --locked -- -D warnings
 
+  tests:
+    name: tests
+    runs-on: ubuntu-latest
+    needs: [changes, lint_fmt]
```

Comment:
More a comment to the reviewer.

By experience it tells me that if you address the potential questions a PR my arise before a reviewer makes his opinion stated, it may lower the friction on a review.